Kickstarter-iOS/Views/Controllers/HelpViewController.swift

```diff
@@ -38,6 +38,7 @@ final class HelpViewController: UIViewController {
 
     _ = tableView
       |> settingsTableViewStyle
+      |> settingsTableViewSeparatorStyle
```

Comment:
Could we add the separator style to the tableView style instead?
```
public let settingsTableViewStyle = UITableView.lens.backgroundColor .~ .ksr_grey_200
  <> settingsTableViewSeparatorStyle
```

Reply:
We can't do that quite yet because the settings newsletters screen is using the `settingsTableViewStyle` but still using custom separators. I thought it might get a little complicated to remove them on that screen in this PR, but once we do we can merge the two styles.